<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeaderBoardWindow</class>
 <widget class="QDialog" name="LeaderBoardWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="leaderboardText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>15</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold SemiConden</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Leaderboard</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>69</y>
     <width>321</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7. </string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5. </string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8. </string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>10. </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color: #CD7F32;
}</string>
      </property>
      <property name="text">
       <string>3. </string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4. </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color: #FFDF00;
}</string>
      </property>
      <property name="text">
       <string>1. </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color: #C0C0C0;
}</string>
      </property>
      <property name="text">
       <string>2. </string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6. </string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9. </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="Place_1">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color: #FFDF00;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="Place_2">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color: #C0C0C0;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="Place_3">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color: #CD7F32;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="Place_4">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="Place_5">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="Place_6">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="Place_7">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="Place_8">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="Place_9">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="Place_10">
      <property name="font">
       <font>
        <family>Bahnschrift Light SemiCondensed</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
